api.go

```diff
@@ -208,70 +260,14 @@ type Conversation struct {
 	HasThinkingContent bool
 }
 
-// NewConversation creates a new conversation with the given system prompt. It
```

Comment:
Moving this to the bottom really impacts readability, this should be moved back to here.